### PR TITLE
support crowbar node upgrade

### PR DESCRIPTION
**Pull request by jdsn**
during cloudupgrade step also trigger the nodeupgrade
with this also the testsetup step after cloudupgrade will be possible

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -4103,6 +4108,17 @@ function onadmin_crowbarrestore()
...
 }
 
+function onadmin_crowbar_nodeupgrade()
+{
+    # FIXME: change path to "*.json" when crowbar supports it:
+    curl --silent -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/nodes
+    wait_for 360 10 "crowbar_nodeupgrade_status | grep -q '\"left\": *0'" "crowbar to finish the nodeupgrade"
+    if ! crowbar_nodeupgrade_status | grep -q '"failed": *0' ; then
+            crowbar_nodeupgrade_status
+            complain 38 "Crowbar nodeupgrade failed."
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: there is too much indention :)

**Comment by dirkmueller** [comment]
LGTM other than the indentation issue

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: uuups, right

**Comment by jdsn** [comment]
I will wait for the gating job to finish, then force push the indentation change and reset the gating status on the latest commit, as the code does not change due to indentation fixing.

**Comment by bmwiedemann** [comment]
+1

**Comment by jdsn** [comment]
gating still not getting green: my proposal, get this merged to make the failures visible in jenkins (fails in reapplying proposals)

**Comment by rsalevsky** [comment]
+1

**Pull request merged**